packages/leancode_lint/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 9.0.0
+- Disable lints deprecated in Dart 3.3.0
+- Bump minimum Dart version to 3.3
+
 # 8.1.0
```

Comment:
This was never released actually, and it should've been in a major anyway. Could you merge those changelogs please?

Reply:
Yup :+1: